### DISCUSSION TITLE
Demo

### DISCUSSION THREAD

**Discussion by ashwin31** [fastlane/fastlane#17881] [General] [answered]
Can you make demo video of fast lane basic setup for android and iOS deployments.

native and hybrid apps like flutter and react-native.

**Comment by rogerluan** [comment, marked as answer]
Hey @ashwin31 👋 

Unfortunately there's no official video demo to teach users how to set up fastlane. However, I see value in that and if you're willing to record one, that'd be 💯 😊 

For now, you can check our official docs here: https://docs.fastlane.tools
For Android setup: https://docs.fastlane.tools/getting-started/ios/setup/
For iOS setup: https://docs.fastlane.tools/getting-started/ios/setup/

Hope this helps! Cheers!
